client/components/jetpack-colophon/index.jsx

```diff
@@ -4,10 +4,16 @@ import JetpackLogo from 'calypso/components/jetpack-logo';
 
 import './style.scss';
 
-const JetpackColophon = ( { className, translate } ) => {
+const JetpackColophon = ( { className, translate, grayscale = false } ) => {
 	return (
 		<div className={ classNames( 'jetpack-colophon', className ) }>
-			<span className="jetpack-colophon__power">
+			<span
+				className={
+					grayscale
```

Comment:
```suggestion
					classNames( 'jetpack-colophon__power', {
					    'jetpack-colophon__grayscale': grayscale
					} )
```

We could use the existing utility `classNames` to consolidate class names.

Reply:
I'll add this in. 👍